db/migrate/20160217192112_create_rentals.rb

```diff
@@ -6,6 +6,7 @@ def change
       t.integer :department_id, null: false
       t.integer :reservation_id, null: false
       t.references :item_type, null: false
+      t.references :item, null: false
```

Comment:
⚠️  This is worth discussing! Changing old migrations can have some pretty serious ramifications! Luckily, we are not in production, but 'future you' will high five your 'past self' for knowing it.